src/storage_tiering.cpp

```diff
@@ -992,7 +991,9 @@ namespace irods {
         }
 
         if (const auto ec = rcModAVUMetadata(_comm, &set_op); ec < 0) {
-            THROW(ec, fmt::format("failed to set migration scheduled flag for [{}]", _object_path));
+            const auto msg = fmt::format("{}: failed to set migration scheduled flag for [{}]", __func__, _object_path);
+            rodsLog(LOG_ERROR, msg.c_str());
```

Comment:
New logger?

Or open an issue to replace all use of `rodsLog` in the project.

Reply:
Probably this one... https://github.com/irods/irods_capability_storage_tiering/issues/244

Since this is a new instance, should probably not use the old logger. Do you have a preference between the `server` or `rule_engine` category until we introduce a new category just for this plugin?

Reply:
`rule_engine` is the category to use. It is used by the other REPs (that have been updated).

Reply:
Done